### DISCUSSION TITLE
How to completely remove organ/sampleset?

### DISCUSSION THREAD

**Discussion by jason-barba** [GrandOrgue/grandorgue#1571] [Help]
I have installed BureaChurch.orgue on a GO version that was to old for the sample set to work; I only got the basic version to function. After update to latest GO version I of course want also the extended version to function. I have tried to remove the sample set completely so many times now without success so I do no longer know what I am doing...
My problem is that only the basic version is seen in the Load menu, no matter what I do to remove it. Settings/Delete doesn't help. Restarting doesn't help. Settings/Delete cache make no difference in the folder GrandOurgueCache or GrandOrgueData. Likewise no change in folder content if I hit Settings/Delete Preset(s).
What exactly is the procedure to make GO forget everything about a sample set?
See also [#915](https://github.com/GrandOrgue/grandorgue/discussions/915) and [#1457](https://github.com/GrandOrgue/grandorgue/discussions/1457) where this matter has been discussed earlier.

**Comment by kerkovits** [comment]
File→Settings→Organs→The organ you want to delete→Delete cache, delete settings, delete AND press OK. Any deletion takes effect only after pressing OK.

If there are still traces, you can search for the organ hash written in this panel on the filesystem and delete manually

**Reply by jason-barba** [reply]
Tried that once more. After hitting OK an error message pops up something about GrandOrgue not responding. Hit Wait and a new popup showed. Repeated this a couple of times, gave up and hit Quit. Restarted GO and the basic version only was found in Load menu.
Repeated all above. But this time a had to do something else so I left the message about GrandOrgue not responding. When I came back, that pop up had disappeared and Behold! now both the basic and the extended versions of Bureå Church was present under Load.
Alas a little patience was what was required.
Thanks!

**Comment by krasnit** [comment]
There is no Delete settings option in the (latest)  version of GO I have, so all the settings for the organ remain even when the organ is supposedly deleted..

**Comment by oleg68** [comment]
<img width="700" height="845" alt="del" src="https://github.com/user-attachments/assets/57f4ffb9-bfe4-40d5-b22c-d22dfcc081a3" />
